### PR TITLE
Warnings and Lint

### PR DESCRIPTION
**Pull request by bluemellophone**
This PR fixes the various linting errors and some warnings by pytest.

I've also updated the Flake8 ignore list:

```
D100 - Missing docstring in public module
D101 - Missing docstring in public class
D102 - Missing docstring in public method
D103 - Missing docstring in public function
D105 - Missing docstring in public package
D200 - One-line docstring should fit on one line with quotes
D205 - 1 blank line required between summary line and description
D210 - No whitespaces allowed surrounding docstring text
D400 - First line should end with a period
D401 - First line should be in imperative mood
D403 - First word of the first line should be properly capitalized

E203 - whitespace before ':'
E501 - line too long (82 > 79 characters)

N802 - function name should be lowercase
N803 - argument name should be lowercase
N804 - first argument of a classmethod should be named ‘cls’
N805 - first argument of a method should be named ‘self’
N806 - variable in function should be lowercase

W503 - line break before binary operator
```

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,7 +1,9 @@
 [flake8]
+exclude = .git,.env
 # This section configures `flake8`, the python linting utility.
 # See also https://flake8.pycqa.org/en/latest/user/configuration.html
-ignore = D100,D101,D102,D103,D105,D200,D205,D210,D400,D401,D403,E127,E201,E202,E203,E221,E222,E241,E265,E271,E272,E301,E501,N802,N803,N805,N806,W503
+ignore = D100,D101,D102,D103,D105,D200,D205,D210,D400,D401,D403,E203,E501,N802,N803,N804,N805,N806,W503
```
Comment (mmulich) [inline setup.cfg:5]: Slimming down the list 😌

Comment (bluemellophone) [reply setup.cfg:5]: Yep, we can work on migrating these to all of the repos later

File: wbia/dbio/ingest_ggr.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-#!/usr/bin/env python
+# !/usr/bin/env python
```
Comment (mmulich) [inline wbia/dbio/ingest_ggr.py:2]: Gotta bump this line up and remove the space.

Comment (mmulich) [reply wbia/dbio/ingest_ggr.py:2]: Hmm... turns out this works, so nevermind. 🤯

Comment (bluemellophone) [reply wbia/dbio/ingest_ggr.py:2]: This was resolved (and temp.txt was removed) in c0037fa1aa54cbba35b4335f8c47db2c129c1ef0

**Pull request merged**